python/chronos/src/bigdl/chronos/forecaster/base_forecaster.py

```diff
@@ -83,15 +83,12 @@ def model_creator_orca(config):
                 optimizer = self.optimizer_creator(model, self.optim_config)
                 self.internal = Trainer.compile(model=model, loss=loss,
                                                 optimizer=optimizer)
-            self.onnxruntime_fp32 = None  # onnxruntime session for fp32 precision
-            self.openvino_fp32 = None  # placeholader openvino session for fp32 precision
-            self.jit_fp32 = None  # placeholader jit session for fp32 precision
-            self.onnxruntime_int8 = None  # onnxruntime session for int8 precision
-            self.openvino_int8 = None  # placeholader openvino session for int8 precision
-            self.jit_int8 = None  # placeholader jit session for int8 precision
-            self.pytorch_int8 = None  # pytorch model for int8 precision
-            self.optim_model = None  # accelarated model obtained from .optimize()
-            self.metadata = None  # str indicates model type of optim_model
+
+            self.accelerated_model = None  # accelerated model obtained from various accelerators
+            self.accelerate_method = None  # str indicates current accelerate method
+            self.quantize_models = {}      # quantization models, format: {model_type : model}
```

Comment:
leave `self.quantize_models` temporarily here, we shall change `quantize`'s API later to avoid quantize 2 models at the same time.